ops/roles/ota/github-clone/tasks/main.yml

```diff
@@ -24,15 +27,15 @@
       pull: yes
       args:
         ENV_FILE: '{{ env_file }}'
-      path: '/home/{{ ansible_user }}/{{ app }}'
-    name: '{{ app }}:{{ result_git_clone.before }}'
+      path: '/home/{{ ansible_user }}/{{ name }}'
+    name: '{{ name }}:{{ git_clone_result.after[:7] }}'
```

Comment:
What is `after[:7]`?

Reply:
This is to keep only the first 7 characters of the commit SHA

Reply:
Okay thanks, maybe add a comment

Reply:
Sorry I did not see it was already merged